packages/ckeditor5-utils/src/index.ts

```diff
@@ -93,6 +93,8 @@ export { default as uid } from './uid.js';
 export { default as delay, type DelayedFunc } from './delay.js';
 export { default as verifyLicense } from './verifylicense.js';
 export { default as wait } from './wait.js';
+export { default as sanitizeHTML } from './sanitizeHTML.js';
```

Comment:
This is not related to the PR.

Reply:
Related to second PR in commercial repo, updated description.